gym/dev_wrappers/torch_wrapper.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional, Tuple, Union
+
+import torch
+from brax.io.torch import jax_to_torch, torch_to_jax
+
+from gym import Env, Wrapper
+
+
+class TorchWrapper(Wrapper):
```

Comment:
Minor point that I may be wrong on, `to_torch_v0`, most of our wrappers don't say wrapper in their class, that is inferred. Plus the underscore style is a change we are going for in the new wrappers as we allow the version number in case we mess up and need to change something critical.  
Same for the tensorflow wrapper, `to_tensorflow_v0`. 
I feel there could be a better name, I can't think of one now